cmd/sonobuoy/app/worker.go

```diff
@@ -65,10 +66,6 @@ var singleNodeCmd = &cobra.Command{
 	Args:  cobra.ExactArgs(0),
 }
 
-func runGatherHelp(cmd *cobra.Command, args []string) {
-	cmd.Help()
-}
```

Comment:
No need to add this, if you don't set a `Run` or `RunE` it will show usage messages already.